pkg/apis/pipeline/v1alpha1/pipelinerun_types.go

```diff
@@ -74,6 +74,17 @@ type PipelineRunSpec struct {
 
 	// PodTemplate holds pod specific configuration
 	PodTemplate PodTemplate `json:"podTemplate,omitempty"`
+
+	// ExpirationSecondsTTL limits the lifetime of a PipelineRun that has finished
+	// execution (either Succeeded or Failed). If this field is set,
+	// ExpirationSecondsTTL after the PipelineRun finishes, it is eligible to be
+	// automatically deleted. When the PipelineRun is being deleted, its lifecycle
+	// guarantees (e.g. finalizers) will be honored. If this field is unset,
+	// the PipelineRun won't be automatically deleted. If this field is set to zero,
+	// the PipelineRun becomes eligible to be deleted immediately after it finishes.
+	// This field is alpha-level.
```

Comment:
I'm not sure we need so specify this. The whole API is alpha level right now.

Reply:
Oh, I'll delete it